vumi/utils.py

```diff
@@ -13,12 +13,14 @@
 from twisted.internet import protocol
 from twisted.internet.defer import succeed
 from twisted.python.failure import Failure
-from twisted.web.client import Agent, ResponseDone, WebClientContextFactory
+from twisted.web.client import Agent, ResponseDone
 from twisted.web.server import Site
 from twisted.web.http_headers import Headers
 from twisted.web.iweb import IBodyProducer
 from twisted.web.http import PotentialDataLoss
 from twisted.web.resource import Resource
+from treq._utils import default_pool, default_reactor
```

Comment:
I don't think it's a good idea to import from a private module.. These methods could change.